tests/test_dataset.py

```diff
@@ -295,60 +295,37 @@ def setup_dataset(self):
                               'dataset_name': 'None', 'dataset_type': 'None'}
         )
 
-    @patch('analysis_driver.dataset.clarity.get_run')
-    def test_rapid_samples_by_lane(self, mocked_get_run):
-        mocked_get_run.return_value = MockedRunProcess(container=mocked_flowcell_pooling)
-        assert self.dataset.rapid_samples_by_lane == {}
+    def test_rapid_samples_by_lane(self):
+        self.dataset._run_status = mocked_run_status
+        with patch('egcg_core.rest_communication.get_document', return_value={}):
+            assert self.dataset.rapid_samples_by_lane == {}
+        responses = [{},
+                     {'sample_id': 'sample2', 'Rapid Analysis': 'Yes', 'project_id': '10015AT'},
+                     {}, {}, {}, {}, {}, {}]
+        with patch('egcg_core.rest_communication.get_document', side_effect=responses):
+            self.dataset._rapid_samples_by_lane = None
+            assert self.dataset.rapid_samples_by_lane == {'2': {'sample_id': 'sample2', 'Rapid Analysis': 'Yes', 'project_id': '10015AT'}}
 
-        self.dataset._rapid_samples_by_lane = None
```

Comment:
I think this now conflicts since I merged #420

Reply:
I've merged the release branch and resolved conflicts. This one did not show up.